styles.css

```diff
@@ -0,0 +1,116 @@
+/* styles.css */
+
+body {
+  background-color: hsl(30, 54%, 90%);
+  font-family: "Outfit", sans-serif;
+  color: hsl(30, 10%, 34%);
+}
+
+.pageContent {
+  width: 100%; 
+  margin: auto;
+  background-color: hsl(0, 0%, 100%);
+  padding: 3%;
+  border-radius: 15px;
+}
+
+img {
+  width: 100%;
+  border-radius: 15px;
+}
+
+figure {
+  width: 100%;
+  margin: auto;
+}
+
+h1 {
+  color: hsl(24, 5%, 18%);
+}
+
+h1,
+h2 {
+  font-family: "Young Serif";
+  font-weight: 400;
+}
+
+p {
+  font-size: 16px;
+}
+
+strong {
+  color: hsl(30, 10%, 34%);
+}
+
+.preparation {
+  background-color: hsl(330, 100%, 98%);
+  padding-left: 3%;
+  padding-top: 1%;
+  padding-bottom: 1%;
+  border-radius: 2%;
+}
+
+h3 {
+  color: hsl(332, 51%, 32%);
+  font-weight: 600;
+}
+
+h2 {
+  color: hsl(14, 45%, 36%);
+}
+
+ol,
+.quantity {
+  color: hsl(14, 45%, 36%);
+  font-weight: 800;
+}
+
+.listContent {
+  color: hsl(30, 10%, 34%);
+  font-weight: 400;
+}
+
+li {
+  padding-left: 3%;
+  margin-bottom: 3%;
```

Comment:
why you are using percentages? what are the advantages and disadvantages of it ?

Reply:
Percentages offer flexibility and responsiveness in layout design, but they not always provide the level of precision required for certain design scenarios.